packages/wdio-allure-reporter/src/index.js

```diff
@@ -259,16 +259,29 @@ class AllureReporter extends WDIOReporter {
         }
     }
 
-    addStep({step}) {
+    startStep(title) {
         if (!this.isAnyTestRunning()) {
             return false
         }
+        this.allure.startStep(title)
+    }
 
-        this.allure.startStep(step.title)
+    endStep(status) {
+        if (!this.isAnyTestRunning()) {
+            return false
+        }
+        this.allure.endStep(status)
+    }
+
+    addStep({step}) {
+        if (!this.isAnyTestRunning()) {
```

Comment:
Is it necessary here? Because next first function `this.startStep` has the same inside

Reply:
To be honest I'm not really sure. I didn't change behaviour of existing steps, only added  startStep and endStep.

If I simply remove this verification  corresponding test will fail because it expects `false` as a return value while we are returning `undefined`.

Reply:
If you are sure that returning undefined instead of false in this case is fine, I can do the change.